### PR TITLE
Greatly building on marketing content

### PR DESCRIPTION
**Pull request by AsafGerchak**
Basically killed it dead, totally murdered it, wow so much death oh god oh no death is everywhere.

Changes in this PR:
- Added tagline
- Looked handsome

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,6 +10,7 @@
 <body>
 
   <h1>Shuper Shpecial Webshite</h1>
+  <h4>That'sh a great webshervish</h4>
```
Comment (owenkcraig) [inline index.html:13]: Hey Safi. You should've made this an H2 😵‍💫

File: index.html [hunk 2]
```diff
@@ -10,6 +10,7 @@
 <body>
 
   <h1>Shuper Shpecial Webshite</h1>
```
Comment (owenkcraig) [inline index.html:12]: Love this.

**Review by owenkcraig — CHANGES_REQUESTED** [review]
Looking very good, but we've got to get that H4 out of there

File: index.html (hunk 2 above)
Comment (AsafGerchak) [reply index.html:12]: Thanks! I actually didn't do this, that was already there, but I am happy to take credit. Wow I nailed it!

**Review by owenkcraig — APPROVED** [review]
That's a spicy PR! Approved!

**Pull request merged**